Sources/Recycling/ListViews/SavedSearches/Cell/SavedSearchesListViewCell.swift

```diff
@@ -0,0 +1,104 @@
+        view.axis = .vertical
+        view.addArrangedSubview(self.titleLabel)
+        view.addArrangedSubview(self.subtitleLabel)
+
+        return view
+    }()
+
+    @objc lazy var settingsButton: UIButton = {
+        let button = UIButton(type: .custom)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.setImage(UIImage(named: FinniversImageAsset.settings), for: .normal)
+        button.addTarget(self, action: #selector(showSettings(button:)), for: .touchUpInside)
+        button.isAccessibilityElement = true
+
+        return button
+    }()
+
+    @objc lazy var arrowImageView: UIImageView = {
+        let view = UIImageView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.contentMode = .center
+        view.image = UIImage(named: FinniversImageAsset.arrowRight)
+
+        return view
+    }()
+
+    override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+        contentView.addSubview(stackView)
+        contentView.addSubview(settingsButton)
+        contentView.addSubview(arrowImageView)
+
+        let margin: CGFloat = 8
+        let settingsSize: CGFloat = 50
+        let arrowSize: CGFloat = 20
+        NSLayoutConstraint.activate([
+            stackView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: margin),
+            stackView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -margin),
+            stackView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: margin * 2),
+            stackView.trailingAnchor.constraint(equalTo: settingsButton.leadingAnchor, constant: -margin * 2),
+
+            settingsButton.widthAnchor.constraint(equalToConstant: settingsSize),
+            settingsButton.heightAnchor.constraint(equalToConstant: settingsSize),
+            settingsButton.centerYAnchor.constraint(equalTo: contentView.centerYAnchor),
+            settingsButton.trailingAnchor.constraint(equalTo: arrowImageView.leadingAnchor),
+
+            arrowImageView.widthAnchor.constraint(equalToConstant: arrowSize),
+            arrowImageView.heightAnchor.constraint(equalToConstant: arrowSize),
+            arrowImageView.centerYAnchor.constraint(equalTo: contentView.centerYAnchor),
+
+            // We use "greaterThanOrEqualTo" instead of "equalTo" because otherwise we get an AutoLayout error
+            arrowImageView.trailingAnchor.constraint(greaterThanOrEqualTo: contentView.trailingAnchor, constant: -margin * 2)
+            ])
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    @objc func showSettings(button: UIButton) {
+        delegate?.savedSearchesListViewCell(self, didPressSettingsButton: button)
+    }
+
+    // MARK: - Dependency injection
+
+    public var model: SavedSearchesListViewModel? {
```

Comment:
There isn't much reason for the cell to keep a reference to the model is it? This could then just be a func to set values to show. But no big deal :)

Reply:
I agree but I think it's better to be consistent instead of being pragmatic when it comes to building the components makes it quicker to make decisions and also to understand code if all is build in the "same" way.

Reply:
Yes, if it is a consistent pattern it is good to keep doing it the same everywhere 👍